noxfile.py

```diff
@@ -0,0 +1,8 @@
+from nox import session
+
+
+@session(reuse_venv=True)
+def docs(session):
+    session.install("-e.")
+    session.install("-rdocs/requirements.txt")
+    session.run("sphinx-build", "docs", "_build/html")
```

Comment:
What's the difference between the previous build workflow and this one?

Reply:
sorry i missed that description. i added instructions to `docs/README.md` for building the docs. you'll need `nox`, `pip install nox`. the run the `nox` `docs` session as:

     nox -s docs

`nox` is a tool to ensure that `sphinx` builds in its own virtual environment, and doesn't effect the developer's primary environments.  its specifically good for something like `jupyter-book` that has a lot of dependencies that are better isolated from other environments. also, it provides a convenient cli that can work locally/ci.